wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/Options.new.kt

```diff
@@ -42,7 +42,7 @@ class Options(
 
   fun retainLinked() = Options(optionType, emptyList())
 
-  operator fun get(protoMember: ProtoMember): Any? {
```

Comment:
Is this change related to this PR? If it's not, I'd prefer to do it in a separated PR.